crates/test-cluster/src/lib.rs

```diff
@@ -1058,3 +1058,24 @@ impl Default for TestClusterBuilder {
         Self::new()
     }
 }
+
+pub async fn fund_address_and_return_gas(
+    context: &WalletContext,
```

Comment:
Could this be a method of TestCluster? If so, it's generally preferred for discoverability. Global functions are really hard to discover.